---
title: 'The Identification of Bacteria on Individual versus Communal PC Keyboards'
author: "Jacinda Chen"
date: "October 6, 2019"
output: word_document
csl: bioinformatics.csl
bibliography: references.bib
---

```{r setup, include = FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

# Introduction

Computers have become an essential part of college life as a result of the increased need of the internet. Food and drinks are constantly consumed around keyboards, which might provide food for whatever is living on top of the keyboard, if there is [@malik2014study]. Therefore, more and more students are required to buy or have access to computers. Keyboards are being touched constantly in personal laptops as well as shared computers that the university offers and most people are not aware of the number of microorganisms present on their keyboards [@al2011bacterial]. I will be comparing the bacteria found on the keyboards of individual students' laptops versus the keyboards of shared computers on the USF campus. Previous research done on this topic has found that shared computers had more microorganisms than individual computers [@anderson2009microbial]. This project will circle around hygiene and aim to figure out if shared computers should be cleaned more often or if personal laptops are not as sanitary as they seem.  

# Methods

## Study design

### Bacterial Collection

The surface samples were collected during the lab section of the Bioinformatics class on September 3rd, 2019. In total, twelve tubes were collected (two tubes per sample in four different locations). For each location, two different keys on the keyboard was swabbed. Each sterile swab was dipped in sterile PBS (Phosphate Buffer Saline at pH 7.4)to moisten, then wrung of excess PBS against the side of the tube. The surface was then swabbed for 15 seconds. The tip of the swab was broken off into a labelled eppendorf with initials, date,and an unique sample ID. Samples were gathered first in the first floor of Toler Hall, where one sample was gathered for the "F" key and another sample was gathered for the "F" key on the shared computer at the front desk and put into labelled eppendorfs. On the fourth floor of the University Center, specifically in the graphics center on a shared computer, samples were gathered for the "F" key. On the first floor of the library, where samples were gathered on a shared computer, samples were once again gathered and put into labelled eppendorfs for the "F" key. Inside the Bioinformatics class, samples were gathered from three different laptops from three different students. Student A's samples were gathered and inserted into labelled eppendorfs for the "F" key, Student B's samples were gathered and inserted into labelled eppendorfs for the "F" key, and Student C's samples were gathered and inserted into labelled eppendorfs for the "F" key.

### Dilution Series

The dilution series was done with the "F" key group. In order to prepare the undiluted group, I put 200 microliters of PBS into the six eppendorfs with swabs. These six eppendorfs were then vortexed for fifteen seconds. There were another six eppendorfs that were dedicated to 1:10 dilutions of the original six eppendorfs. For these six 1:10 dilution eppendorfs, I added 180 microliters of PBS to each of them. Twenty microliters of the one undiluted eppendorf was then added to its respective 1:10 dilution eppendorf to complete the dilution. This was then repeated another five times to complete the other 1:10 dilutions. I vortexed these six eppendorfs for five seconds. In order to do the 1:100 dilutions, I once again added 180 microliters of PBS into each of the six eppendorfs labelled respectively to the undiluted and the 1:10 dilution series. Twenty microliters of a 1:10 dilution was then added to its respectively labelled 1:100 dilution eppendorf. I repeated this step for the other five 1:100 dilution eppendorfs. The six 1:100 dilution eppendorfs were then vortexed for 5 seconds.

### Plating "F" Key Group

I used 18 100 mm TSA(Tryptic soy agar) plates for plating samples from each of my 2 treatments (n = 3 per group, 6 total). I put 100 microliters of an eppendorf from the undiluted series onto a TSA plate and used sterile rattler Plating Beads (5 mm) to spread the medium evenly across the TSA plate. I repeated this step for the other five eppendorfs from the undiluted series, the six eppendorfs from the 1:10 dilution series, as well as six eppendorfs from the 1:100 dilution series. These 18 TSA plates were then incubated at 37^o Celcius for a week.

### DNA Extraction

I followed the manufacturer's protocal from the Sigma REDExtract-N-Amp kit. *cite this kit* I labelled a 1.5 mL tube for each sample and added cells from one colony to the tube using a sterile peppette tip (not pipetting). One colony was chosen each from the TSA plate of the undiluted sample from Toler Hall, the University Center, the library, and Student B. Two colonies were chosen from the TSA plate of the undiluted sample from Student C. I added 100 microliters of extraction solution then vortexed the tube for 60 seconds. This was repeated for the other five 1.5 mL tubes. All six tubes were incubated at 95^o Celcius for ten minutes then vortexed for 60 seconds. I added 100 microliters of dilution solution to each of the six tubes and vortexed them for five seconds. These six tubes were then centrifuged for five minutes at 14,000 rpm. For the Qubit protocal, I added 198 microliters of the Qubit solution to the qubit tube. Two microliters of the DNA extraction was added to the 198 microliters of Qubit solution. This step was repeated for the other five DNA extractions. I vortexed all six Qubit tubes for five seconds then incubated them for five minutes in the dark. The Qubit machine was used to measure the concentration of DNA. 

### Touchdown PCR for Cultured "F" key series

I labelled a tube for the master mix. Inside, I put 77 µL of Amp, 6.2 µL of 27f primer, 6.2 µL of 1492r primer, 7.7 µL BSA, and 49.5 µL of water. These volumes of the reagents were calculated according to (n+1) + 10%(n+1) where n is the number of samples to amplify with PCR. The master mix was vortexed for three seocnds and centrifuged at low speed. I aliquoted 19 µL of the master mix to each 250 µL sample tube, six in total, and one other 250 µL tube for the negative control. One µL of the sample was added to each of the six sample tubes. One µL of water was added for the negative control. The seven 250 µL PCR strip tubes were put in specific PCR cycles and conditions. The PCR strip tubes were put in the machine for 95^o C for 5 minutes. Then, they were put in the machine for 94^o C for 30 seconds, 65^o C for 30 seconds, 72^o C for 1 minute. These last three times were rpeeated for ten cycles, but the temperature was decreased by one degree per cycle. The next three temperatures and times were 94^o C for 30 seconds, 55^o C for 30 seconds, 72^o C 1 minute. These last three times were repeated for 25 cycles. Lastly, the PCR sample tubes went through 72^o C for 10 minutes and held at 4^o C. The electrophoresis gel was made with 2% agarose gel, 140 V for 30 minutes, SYBR safe dye, and 1x TAE buffer. An Ez load 1 kb Molecular Ruler (already has glyceral, dye, and DNA set fragments), or a ladder, was added to the first and last wells of both lines of wells in the gel. One µL of each of my seven PCR tubes was pipetted into seven wells of the gel. After checking for successful amplification using gel electrophoresis, PCR products were cleaned with ExoSAP (Invitrogen) and sent for unidirectional Sanger sequencing at MCLAB (South San Francisco, CA) using the 27f primer. 

### Geneious Prime

After downloading the acb files into Geneious Prime, my first task was to assess whether or not the sequences were usable. Then, I had to trim or correct usable sequences. This was done by trimming off a large part of the beginning and ending of the sequences (since they were filled with low quality or unusable bases) and going through the low quality  bases in the middle of the sequence to decide whether or not they should be deleted, changed, or an ambiguity code needed to be added from the IUPAC Ambiguity Codes. 

## Analysis

Bacterial Collection: For each location, two different keys on the keyboard was swabbed because there was not enough surface area on one key to be swabbed twice. 

Dilution Series: The samples were put into the group key "F". The dilution series was done with the "F" key group. Since it was not known what type of bacteria it was nor how much of it would grow, it was necessary to dilute the samples in order to maximize the number of environments that the bacteria could possibly grow in. The undiluted series were vortexed for 15 seconds in order to get the bacteria off the swabs and into the PBS. *Talk about why vortex* 

DNA Extraction: *Be careful not to grab agar.* Two colonies were chosen from the TSA plate of the undiluted sample from Student C and none from Student A because Student A did not have any visible colonies growing on the TSA plate. The purpose of adding the caustic extraction solution was because it broke down cell walls. Incubated at 95^o Celcius to break membranes. Dilution solution was added to stop the extraction solution from digesting DNA too. The dilution solution should be added in under twenty minutes from when the extraction solution was first added. The last vortex for five seconds was to mix the extraction and dilution solutions. Make sure the centrifuge is balanced andto put on the lid. Qubit *incubate dark because...* It was important to measure the concentration of DNA in order to ensure that we had enough DNA to amplify in PCR, a later step.

Touchdown PCR for "F" key series: Why use touchdown PCR. The PCR strip tubes were put in the machine for 95^o C for 5 minutes as the initial denaturing step where the temperature was slightly higher and the time was slightly longer than normal. Then, they were put in the machine for 94^o C for 30 seconds for more denaturing, 65^o C for 30 seconds for taq polymerase extension as well as to anneal primers to DNA (this temperature is critical to pull the G/C and A/T Hydrogen bonds apart and cool enough to let the primers anneal properly), and 72^o C for 1 minute for the polymerase to make copies that used dNTPS. These last three times were rpeeated for ten cycles, but the temperature was decreased by one degree per cycle. The touchdown PCR was needed to change selectivity and to replicate slower so that more attractions would click together at lower temperatures. The purpose of starting at a higher temperature was to be more selective at the first round of 65^o C with the copies of the best match. As the temperature decreased by one degree, it created matches and copies preferentially of the already selective primer rather than the original strand. Also, if i had accidentally put two colonies instead of one, one colony would most likely be a better match to the primer, then the primer would preferentially amplify that one. The next three temperatures and times were 94^o C for 30 seconds to denature (it is shorter than the five minutes used in the beginning because DNA is double-stranded and have a lot of hydrogen bonds to break), 55^o C for 30 seconds to anneal, 72^o C 1 minute to extend. These last three times were repeated for 25 cycles. Lastly, the PCR sample tubes went through 72^o C for 10 minutes and held at 4^o C. 

Gel Electrophoresis: For the gel electrophoresis, buffer was used instead of water because nothing would happen with water. Buffer conducts a high level of current. Agarose gel was used rather than poyacrymyli because agarose is less dense and gives DNA more space to stretch out. The master mix contained glyceral, which made my sample more dense than water, causing it to sink at the bottom of the well in the gel instead of mixing with water in the gel. The dye was used to make our samples more visible. DNA molecules are negatively charged, so they are pulled sideways towards the positive current. There were ladders in the first and last wells of each of the two lines of wells in order for them to act as standards (a bunch of fragments of DNA at known lengths). The ladders allow us to compare unknowns to known lengths of DNA fragments as well as different concentrations, which is measured by how bright the bands are. It also acts as the positive control from the cyberworking. If the electrophoresis and the cyber works then you should see the known lengths of DNA fragments. The ladder looks more grouped together above 3 kb bcause the gel only ran for 30 minutes.

Geneious Prime: Knowing that I was sequencing bacteria, some of the ambiguous peaks I observed was probably because I might have sampled two colonies instead of one. Another reason why this may have occurred was because the colony sampled is an entire population, which consists of millions of cells that have divided and might have had mutations. Ideally, I had over 200-300 bases for my project.

# Results

## Culture Data

```{r culture-abundances-1x}
# Make a boxplot of undiluted culture abundances
culture_data <- read.csv("data")
```

## Table of Qubit DNA extraction samples of Key "F" series (cultured)

| Sample | Qubit Number Range (ng/µL) |
|--------|----------------------------|
| 1F     | 5.40                       |
| 2F     | 6.10                       |
| 3F     | 8.76                       |
| 5F     | 7.51                       |
| 6Fo    | 5.66                       |
| 6Fp    | 9.05                       |



# Discussion

# Sources Cited
