---
title: 'The Identification of Bacteria on Individual versus Communal PC Keyboards'
author: "Jacinda Chen"
date: "October 6, 2019"
output: word_document
csl: bioinformatics.csl
bibliography: references.bib
---

```{r setup, include = FALSE}
knitr::opts_chunk$set(echo = FALSE, message = FALSE, warning = FALSE)
```

# Introduction

Computers have become an essential part of college life as a result of the increased dependency on the internet. Therefore, an increasing amount of students are required to buy or have access to computers. Food and drinks are constantly consumed around keyboards, which provide living resources for whatever is living on top of the keyboard, if there is [@malik2014study]. Keyboards are being touched constantly in personal laptops as well as shared computers open to students, faculty and staff (that the university offers). Most people are not aware of the number of microorganisms present on their keyboards [@al2011bacterial]. My primary question was whether or not the keyboards of individual USF college students' laptops from the Bioinformatics class have more or less microbacterial species compared to shared computers on campus from the library, the UC and Toler Hall. Previous research done on this topic has found that shared computers had more microorganisms than individual computers [@anderson2009microbial]. This project will circle around hygiene and aim to figure out if  computers should be cleaned more often and if it would be necessary to raise awareness of keyboard hygiene as most people are not that informed about the number of the microorganisms present on keyboards. It is recommended that food and drink should be seperated from computers and keyboards should be disinfected regularly [@hamzeh2011computer]. This is a problem, especially on a school campus where diseases spread quickly. In the interest of people trying to find ways to prevent the spread of these microorganisms, it is important to keep regularly used computers clean, especially in shared public settings. Although the microorganisms found on computers may not be pathogenic or malignant for most people, they can definitely cause harm for people with compromised immune systems [@fraser2009bacterial]. An example of this could be burn victims, where the issue was raised if computer keyboard contamination was a possible method of transmission of methicillin-resistant *Staphylococcus aurerus* onto patients [@isaacs1998swabbing]. 

Even though different institutions and people can have various methods of cleaning keyboards, without a regular and frequent keyboard cleaning, bacteria is free to grow. The longer that keyboards are not cleaned, the higher the chance that users of the keyboards will contract diseases from the microorganism or transmit the microorganism elsewhere. The microorganisms with resistances to most antibiotics or disinfectants are growing steadily and exponentially, which allows a growing amount of microorganisms to have longer lifespans on keyboards. Because of this, patients with compromised immune systems, especially burn victims, no longer have the assurance of antibiotics curing them fully. Most of the bacteria that plague burn victims or other immunosuppressed patients are gram-negative bacteria, which survive significantly longer on hard surfaces (such as polyurethane, or computer keyboard covers) and humid conditions [@neely2000survey]. A longer lifespan gives microorganisms more time to transmit and spread to other places where they have more time to replicate, mutate, and increase survivability [@eltablawy2009microbial]. Food crumbs, soda residues, and more food residues help millions of bacteria grow. All of the sampled keyboards were positive for microorganism growth [@eltablawy2009microbial]. The environmental surface at which bacteria grow on largely affects their survival times.

My hypothesis is that the keyboards of shared computers have more microbacterial species than individual keyboards because of the increased number of people (students, faculty, staff, etc.) that have access to the keyboard. I collected three bacterial samples from three different individual laptops on the "F" key from students from USF (University of San Francisco) and three other samples from three public spaces on campus, also on the "F" key. These bacteria were diluted then grown in cultures with their dilutions, where DNA was extracted from them. Touchdown PCR was done along with Sanger sequencing, whose results I analyzed with Geneious Prime and R Studio. My primary findings for the culture data was a significant difference in the number of morphotypes between Communal and Individual computers. For the molecular data, it was important that the BLAST table agreed with the Maximum Likelihood Phylogeny and the Bayesian Phylogeny in which samples were more closely related. 

# Methods

## Study design

I planned to look at the number of bacteria species on laptop keyboards of individual University of San Francisco students versus the bacteria found on shared computers. I sampled the "F" key for different keyboards of three different laptops, each used by individual USF students, as well as the keyboards of three different shared computers on the main campus of USF (Toler Hall, University Center, and Gleeson Library). My primary question was whether or not the keyboards of individual USF college students' laptops from the Bioinformatics class have more or less microbacterial species compared to shared computers on campus from the library, the UC and Toler Hall. 

I collected bacteria, diluted the samples, plated the samples on TSA plates, extracted DNA from the samples, did touchdown PCR, gel electrophoresis, and Sanger sequencing. These results were analyzed with Geneious Prime and R Studio.

## Bacterial Collection

The surface samples were collected during the lab section of the Bioinformatics class on September 3rd, 2019. In total, twelve tubes were collected (two tubes per sample in four different locations). For each location, two different keys on the keyboard was swabbed. Each sterile swab was dipped in sterile PBS (Phosphate Buffer Saline at pH 7.4)to moisten, then wrung of excess PBS against the side of the tube. The surface was then swabbed for 15 seconds. The tip of the swab was broken off into a labelled eppendorf with initials, date,and an unique sample ID. Samples were gathered first in the first floor of Toler Hall, where one sample was gathered for the "F" key and another sample was gathered for the "F" key on the shared computer at the front desk and put into labelled eppendorfs. On the fourth floor of the University Center, specifically in the graphics center on a shared computer, samples were gathered for the "F" key. On the first floor of Gleeson Library, where samples were gathered on a shared computer, samples were once again gathered and put into labelled eppendorfs for the "F" key. Inside the Bioinformatics class, samples were gathered from three different laptops from three different students (Student A, Student B, and Student C) in the same way for the "F" key.

## Dilution Series

The dilution series was done with the "F" key group. In order to prepare the undiluted group, I put 200 µL of PBS into the six eppendorfs with swabs. These six eppendorfs were then vortexed for fifteen seconds. There were another six eppendorfs that were dedicated to 1:10 dilutions of the original six eppendorfs. For these six 1:10 dilution eppendorfs, I added 180 µL of PBS to each of them. Twenty µL of the one undiluted eppendorf was then added to its respective 1:10 dilution eppendorf to complete the dilution. This was then repeated another five times to complete the other 1:10 dilutions. I vortexed these six eppendorfs for five seconds. In order to do the 1:100 dilutions, I once again added 180 µL of PBS into each of the six eppendorfs labelled respectively to the undiluted and the 1:10 dilution series. Twenty µL of a 1:10 dilution was then added to its respectively labelled 1:100 dilution eppendorf. I repeated this step for the other five 1:100 dilution eppendorfs. The six 1:100 dilution eppendorfs were then vortexed for 5 seconds.

## Plating "F" Key Group

I used 18 100 mm TSA (Tryptic soy agar) plates for plating samples from each of my 2 treatments (n = 3 per group, 6 total). I put 100 µL of an eppendorf from the undiluted series onto a TSA plate and used sterile rattler Plating Beads (5 mm) to spread the medium evenly across the TSA plate. I repeated this step for the other five eppendorfs from the undiluted series, the six eppendorfs from the 1:10 dilution series, as well as six eppendorfs from the 1:100 dilution series. These 18 TSA plates were then incubated at 37^o^ Celcius for a week.

## DNA Extraction

I followed the manufacturer's protocal from the Sigma REDExtract-N-Amp kit ([@weberextract]. I labelled a 1.5 mL tube for each sample and added cells from one colony to the tube using a sterile peppette tip (not pipetting). One colony was chosen each from the TSA plate of the undiluted sample from Toler Hall, the University Center, the library, and Student B. Two colonies were chosen from the TSA plate of the undiluted sample from Student C. I added 100 µL of extraction solution then vortexed the tube for 60 seconds. This was repeated for the other five 1.5 mL tubes. All six tubes were incubated at 95^o^ Celcius for ten minutes then vortexed for 60 seconds. I added 100 µL of dilution solution to each of the six tubes and vortexed them for five seconds. These six tubes were then centrifuged for five minutes at 14,000 rpm. For the Qubit protocal, I added 198 µL of the Qubit solution to the qubit tube. Two µL of the DNA extraction was added to the 198 µL of Qubit solution. This step was repeated for the other five DNA extractions. I vortexed all six Qubit tubes for five seconds then incubated them for five minutes in the dark. The Qubit machine was used to measure the concentration of DNA. 

## Touchdown PCR for Cultured "F" key series

I labelled a tube for the master mix. Inside, I put 77 µL of Amp, 6.2 µL of 27f primer, 6.2 µL of 1492r primer, 7.7 µL BSA, and 49.5 µL of water. These volumes of the reagents were calculated according to (n+1) + 10%(n+1) where n is the number of samples to amplify with PCR. The master mix was vortexed for three seocnds and centrifuged at low speed. I aliquoted 19 µL of the master mix to each 250 µL sample tube, six in total, and one other 250 µL tube for the negative control. One µL of the sample was added to each of the six sample tubes. One µL of water was added for the negative control. The seven 250 µL PCR strip tubes were put in specific PCR cycles and conditions. The PCR strip tubes were put in the machine for 95^o^ C for 5 minutes. Then, they were put in the machine for 94^o^ C for 30 seconds, 65^o^ C for 30 seconds, 72^o^ C for 1 minute. These last three times were rpeeated for ten cycles, but the temperature was decreased by one degree per cycle. The next three temperatures and times were 94^o^ C for 30 seconds, 55^o^ C for 30 seconds, 72^o^ C 1 minute. These last three times were repeated for 25 cycles. Lastly, the PCR sample tubes went through 72^o^ C for 10 minutes and held at 4^o^ C. The electrophoresis gel was made with 2% agarose gel, 140 V for 30 minutes, SYBR safe dye, and 1x TAE buffer. An Ez load 1 kb Molecular Ruler (already has glyceral, dye, and DNA set fragments), or a ladder, was added to the first and last wells of both lines of wells in the gel. One µL of each of my seven PCR tubes was pipetted into seven wells of the gel. After checking for successful amplification using gel electrophoresis, PCR products were cleaned with ExoSAP (Invitrogen) and sent for unidirectional Sanger sequencing at MCLAB (South San Francisco, CA) using the 27f primer. 

## Analysis

### Geneious Prime

After downloading the acb files into Geneious Prime, my first task was to assess whether or not the sequences were usable. Then, I had to trim or correct usable sequences. This was done by trimming off a large part of the beginning and ending of the sequences (since they were filled with low quality or unusable bases) and going through the low quality  bases in the middle of the sequence to decide whether or not they should be deleted, changed, or an ambiguity code needed to be added from the IUPAC Ambiguity Codes. I used MAFFT to create an appropriate multiple sequence alignment with all five of my usable and corrected sequences with the default settings and an offset value of 0.123. A Maximum Likelihood Phylogeny and a Bayesian Phylogeny were created. I used *Thermus aquaticus* as an outgroup by downloading the sequence from GenBank. For the Maximum Likelihood Phylogeny, a substitution model of General Time Reversible (GTR) was used with 100 bootstrap replicates with defaults for all other settings. For the Bayesian Phylogeny, I used a substitution model of GTR, a rate variation of Gamma plus Invariants model, as well as a Burn-in Length of 100,000 with all other settings set to default. I used Nucleotide BLAST (Basic Local Alignment Search Tool) by comparing my nucleotide sequences to the sequence databases, where it also calculated the statistical significance.

### R Studio

I used the packages dplyr and ggplot in order to visualize the data with boxplots. I also tested between different sampling groups using Wilcox statistical tests in order to get the p value.

# Results

```{r load-libraries}
library("ggplot2")
library("dplyr")
library("tidyr")
library("readr")
library("broom")
library("knitr")
```

```{r read-in-culture-data}
# read in the culture data from csv
culture_data <- read_csv(
  "data/raw_data/raw_culture_count_data/BIOL422_culturing_data.csv")
```

## Culture Data

### Grown Cultures

![](data/images/method_images/grown_cultures.jpg)

### Colony Abundance Boxplot

**Figure 1** is a boxplot showing the colony abundances in communal versus individual computers at the 1x dilution level. The colony abundances from the Communal computers had larger number of colonies compared to the Individual computers, which is shown by the maximum number of colonies observed in the Individual computers remaining smaller than the minimum of the colony abundances from the Communal computers. I also observed a bigger range of larger values for the colony abundances of Communal computers where the minimum and maxinum number of colony abundances had a difference of around eight, whereas the differences between the minimum and maxinum number of colony abundances for Individual computers was about one. The colony abundance for Individual computers had a singular abnormally abundant colony depicted by the outlier, just under the 2.5 number of colonies mark.  

```{r filter-and-plot-abundances}
# filter out only my 1x dilutions and make a boxplot
culture_data %>%
  filter(student_name == "Jacinda Chen") %>%
  filter(dilution_1_10_or_100 == "1") %>%
  ggplot(aes(x = sample_group, y = number_of_colonies)) +
  geom_boxplot() +
  ggtitle(
    "Comparison of Colony Abundances Between Communal and Individual Computers"
    , "1x dilution") +
  xlab("Sample Group") +
  ylab("Number of Colonies")
```

**Figure 1:** Boxplot of colony abundances at Communal versus Individual computers, 1x dilution. There was a higher median number of colonies from Communal Computer samples, and the mean number of colonies were significantly different between the two sites (Wilcox p = 0.02104).

### Statistical (Wilcox) test for above boxplot

**Table 1** is a statistical (Wilcox) test to compare the number of colony abundances of Communal versus Individual computers at 1x dilution. The p-value was 0.02. 

```{r abundances-stats}
# run a statistical test to compare the two groups of abundances
culture_data %>%
  filter(student_name == "Jacinda Chen") %>%
  filter(dilution_1_10_or_100 == "1") %>%
  wilcox.test(data = .,
              number_of_colonies ~ sample_group) %>%
  broom::glance(x = .) %>%
  knitr::kable()
```

**Table 1:** The statistical (Wilcox) test to compare the number of colony abundances between Communal and Individual computers, 1x dilution.

### Boxplot showing the number of morphotypes in each of your two treatments (1x Dilution)

I made a boxplot showing the number of morphotypes in Communal versus Individual computers for 1x dilution (Figure 2). Both Communal and Individual computers had an outlier of two morphotypes. The Communal computers had a median, minimum, and maximum of one morphotype. Individual computers had a median and minimum of 0 morphotypes and a maximum of 0.75 morphotypes. 

```{r filter-and-plot-morphotypes}
# Comparison of morphotypes between Communal and Individual computers
culture_data %>%
  filter(student_name == "Jacinda Chen") %>%
  filter(dilution_1_10_or_100 == "1") %>%
  ggplot(aes(x = sample_group, y = as.numeric(number_of_morphotypes))) +
  geom_boxplot() +
  ggtitle(
    "Comparison of Morphotypes Between Communal and Individual Computers"
    , "1x dilution") +
  xlab("Sample Group") +
  ylab("Number of Morphotypes")
```

**Figure 2:** Boxplot showing the number of morphotypes between Communal and Individual computers, 1x dilution. The mean number of morphotypes were not significantly different between the communal and individual computers (Wilcox p = 0.09679).

### Wilcox test for above boxplot

The p-value from **Table 2** is 0.10. This was the result of a statistical test to compare the number of morphotypes in Communal versus Individual computer. 

```{r morphotype_stats}
# run a statistical test to compare number of morphotypes between two groups
culture_data %>%
  filter(student_name == "Jacinda Chen") %>%
  filter(dilution_1_10_or_100 == "1") %>%
  wilcox.test(data = .,
              as.numeric(number_of_morphotypes) ~ sample_group) %>%
  broom::glance(x = .) %>%
  knitr::kable()
```

**Table 2:** The statistical (Wilcox) test to compare the number of morphotypes between Communal and Individual computers, 1x dilution.

## Molecular Data

### Table of Qubit DNA extraction samples of Key "F" series (cultured)

**Table 3** depicts the Qubit numbers of the DNA extractions from the "F" key culture samples. The 1F sample had a Qubit number of 5.40 ng/µL, 2F had 6.10 ng/µL, 3F had 8.76 ng/µL, 5F had 7.51 ng/µL, 6Fo had 5.66 ng/µL, and 6Fp had 9.05 ng/µL.  

| Sample | Qubit Number Range (ng/µL) |
|--------|----------------------------|
| 1F     | 5.40                       |
| 2F     | 6.10                       |
| 3F     | 8.76                       |
| 5F     | 7.51                       |
| 6Fo    | 5.66                       |
| 6Fp    | 9.05                       |

**Table 3:** Table of Qubit DNA extraction samples of Key "F" series (cultured)

### Gel electrophoresis image

**Figure 3** showed bright bands for the 1F, 2F, 3F, 5F, and 6Fp samples. The 6Fo sample showed a faint band. The bands all showed up at a similar place,  using to the ladder. There were no bands at all in the negative control.

![](data/images/gel_images/2019-09-18_BIOL422_Zimmerman_culture_PCRs_Cropped_7.png)

**Figure 3:** Gel image with lanes labeled

### Sequence Usability Table

**Table 4** shows the original filename of my sequences that I received back from Sanger sequencing. The unusable ends were trimmed off and four bases out of the remaining 426 bases for the 1F trimmed sequence were changed. For the 2F trimmed sequence, four bases out of 658 bases were changed. Three bases out of 634 bases for the 3F trimmed sequence were edited with ambiguity codes.  For 5F trimmed sequence, seven bases out of 593 bases were changed to ambuity codes. For the 6Fp trimmed sequence, three bases out of 559 bases were changed.

| Sequence Locus | Original Sequence Filename | Corrected Sequence Filename        | Usability | Length after trimming | Manually corrected/adjusted bases |
|----------------|----------------------------|------------------------------------|-----------|-----------------------|-----------------------------------|
| 16S            | JC_1F_27f_A07.ab1          | JC_1F_27f_A07_JC_cleaned_mixed.ab1 | Usable    | 426 bases             | 4                                 |
| 16S            | JC_2F_27f_B07.ab1          | JC_2F_27f_B07_JC_cleaned_mixed.ab1 | Usable    | 658 bases             | 4                                 |
| 16S            | JC_3F_27f_C07.ab1          | JC_3F_27f_C07_JC_cleaned.ab1       | Usable    | 634 bases             | 3                                 |
| 16S            | JC_5F_27f_D07.ab1          | JC_5F_27f_D07_JC_cleaned.ab1       | Usable    | 593 bases             | 7                                 |
| 16S            | JC_6Fo_27f_E07.ab1         | JC_6Fo_27f_E07_JC_failed.ab1       | Unusable  | N/A                   | N/A                               |
| 16S            | JC_6Fp_27f_F07.ab1         | JC_6Fp_27f_F07_JC_cleaned.ab1      | Usable    | 559 bases             | 3                                  |
| 16S            | JC_Neg_27f_G07.ab1         | JC_Neg_27f_G07_JC_failed.ab1       | Unusable  | N/A                   | N/A                               |

**Table 4:** Table depicting the original filenames, usable trimmed sequences that were corrected and renamed, and details surrounding the latter.

### BLAST Table

For the trimmed sequence of 1F, Staphylococcus epidermidis was found to be the closest match using BLAST; the percent identification is 99.53%, the query cover is 100%, and the E-value is 0.0. The same goes for the following trimmed sequences: Staphylococcus hominis subsp. novobiosepticus was matched with 2F; the percent identification is 99.39%,the query cover is 100%, and the e-value is 0.0. Bacillus subtilis was matched with 3F; the percent identification is 99.53%, the query cover is 100%, and the e-value is 0.0. Bacillus subtilis was matched with 5F; the percent identification is 98.82%, the query cover is 100%, and the e-value is 0.0.  Staphylococcus epidermidis was matched with 6Fp; the percent identification is 99.82%, the query cover is 100%, and the e-value is 0.0 (Table 5; Used BLAST to calculate the statistical significance between matched sequences).

| BLAST filenames                    | Description                                                                                         | Percent Identity | Query cover | Accession  | E-Value |
|------------------------------------|-----------------------------------------------------------------------------------------------------|------------------|-------------|------------|---------|
| JC_1F_27f_A07_JC_cleaned_mixed.ab1 | Staphylococcus epidermidis strain IBK-11 16S ribosomal RNA gene, partial sequence                   | 99.53%           | 100%        | MN428237.1 | 0.0     |
| JC_2F_27f_B07_JC_cleaned_mixed.ab1 | Staphylococcus hominis subsp. novobiosepticus strain IBK-7 16S ribosomal RNA gene, partial sequence | 99.39%           | 100%        | MN428234.1 | 0.0     |
| JC_3F_27f_C07_JC_cleaned.ab1       | Bacillus subtilis strain SR3-30 16S ribosomal RNA gene, partial sequence                            | 99.53%           | 100%        | MN421487.1 | 0.0     |
| JC_5F_27f_D07_JC_cleaned.ab1       | Bacillus subtilis strain SR3-30 16S ribosomal RNA gene, partial sequence                            | 98.82%           | 100%        | MN421487.1 | 0.0     |
| JC_6Fp_27f_F07_JC_cleaned.ab1      | Staphylococcus epidermidis strain LY-2 16S ribosomal RNA gene, partial sequence                     | 99.82%           | 100%        | MH930441.1 | 0.0     |

**Table 5:** Table of nucleotide BLAST results of edited sequences.

### Maximum Likelihood Phylogeny with PhyML

In **Figure 4**, *Thermus aquaticus* was used as the outgroup. I found that sequences 6Fp and 2F are grouped together in a monophyletic clade with 85 bootstrap support value. Sequences 3F and 5F are grouped together in a monophyletic clade with 100 bootstrap support. This was also supported by the Bayesian Phylogeny (Figure 5; Bayesian posterior probability > 95).

![](output/phylogenies/sanger_sequences_phyml_phylogeny_image.png)

**Figure 4:** Maximum Likelihood Phylogeny with PhyML using *Thermus aquaticus* as an outgroup.

### Bayesian Phylogeny with Mr. Bayes

In **Figure 5**, *Thermus aquaticus* was used as the outgroup. I found that sequences 6Fp and 2F are grouped together in a monophyletic clade. Sequences 3F and 5F are grouped together in a monophyletic clade. This was also supported with the Maximum Likelihood Phylogeny in **Figure 4** (Figure 4; ML bootstrap > 80).

![](output/phylogenies/sanger_sequences_mrbayes_sorted_topologies_phylogeny_image.png)

**Figure 5:* Bayesian Phylogeny with Mr. Bayes using *Thermus aquaticus* as an outgroup.

# Discussion

## Bacterial Collection 

For each location, two different keys on the keyboard was swabbed because there was not enough surface area on one key to be swabbed twice. 

## Dilution Series
The samples were put into the group key "F". The dilution series was done with the "F" key group. Since it was not known what type of bacteria it was nor how much of it would grow, it was necessary to dilute the samples in order to maximize the number of environments that the bacteria could possibly grow in. The undiluted series were vortexed for 15 seconds in order to get the bacteria off the swabs and into the PBS. 

## DNA Extraction 

Two colonies were chosen from the TSA plate of the undiluted sample from Student C and none from Student A because Student A did not have any visible colonies growing on the TSA plate *(FIGURE BLANK; showing chosen plates and colonies)*. The purpose of adding the caustic extraction solution was because it broke down cell walls and the incubation at 95^o^ Celcius *SHOWN IN FIGURE BLANK* was to break membranes. Dilution solution was added to stop the extraction solution from digesting DNA too. The dilution solution should be added in under twenty minutes from when the extraction solution was first added or the extraction solution would have started to break down the DNA as well. The last vortex for five seconds was to mix the extraction and dilution solutions. I made sure the centrifuge was balanced and to put on the lid when using it. Qubit samples were incubated in the dark because the Qubit machine is sensitive to the light. It was important to measure the concentration of DNA in order to ensure that we had enough DNA to amplify in PCR, a later step.

## Touchdown PCR for "F" key series

The PCR strip tubes were put in the machine for 95^o^ C for 5 minutes. Then, they were put in the machine for 94^o^ C for 30 seconds, 65^o^ C for 30 seconds, and 72^o^ C for 1 minute. These last three cycles were rpeeated for ten additional cycles, where the temperature was decreased by one degree per cycle. Touchdown PCR was used in order to increase selectivity so that higher quality attractions would click together in the beginning. The purpose of starting at a higher temperature was to be more selective at the first round of 65^o^ C with the copies of the best match. As the temperature decreased by one degree, it created matches and copies preferentially of the already selective primer rather than the original strand. Also, if I had accidentally put two colonies instead of one, one colony would most likely be a better match to the primer, then the primer would preferentially amplify that one. The next three temperatures and times were 94^o^ C for 30 seconds, 55^o^ C for 30 seconds, and 72^o^ C 1 minute. These last three cycles were repeated for 25 additional cycles. Lastly, the PCR sample tubes went through 72^o^ C for 10 minutes and held at 4^o^ C. 

## Gel Electrophoresis

For the gel electrophoresis, buffer was used instead of water because nothing would happen with water while buffer is able to conduct a high level of current. Agarose gel was used rather than polyacrymyli because agarose is less dense and gives DNA more space to stretch out. The master mix contained glyceral, which made my sample more dense than water, causing it to sink at the bottom of the well in the gel instead of mixing with water in the gel.  There were ladders in the first and last wells of each of the two lines of wells in order for them to act as standards (a bunch of fragments of DNA at known lengths). The ladders act as the positive control from the cyber working. If the electrophoresis and the cyber works then you should see the known lengths of DNA fragments. The ladder looks more grouped together above 3 kb bcause the gel only ran for 30 minutes.

## Geneious Prime

Knowing that I was sequencing bacteria, some of the ambiguous peaks I observed was probably because I might have sampled two colonies instead of one. Another reason why this may have occurred was because the colony sampled is an entire population, which consists of millions of cells that have divided and might have had mutations. Ideally, I had over 200-300 bases for my project.

## Colony Abundances and the Statistical test

(Hard to interpret data/takes out human subjectivity when looking for patterns that really aren't there)

## Number of morphotypes and the Statistical test

Each of these two should also be supported with a statistical test for a difference in group means.
The easiest approach for this will be to use a Wilcox test (a nonparametric alternative to the t-test)

## Table of Qubit



## Gel Image



## Accepted Sequences



## BLAST Table

The Query Cover is a number that shows the percentage of how much of the target sequence covers the query sequence. The percent identity measures how similar the two sequences are. A higher percent identity means that the two sequences belong to species that are more closely related. The accession number is an unique tag for a sequence that allows me to track different versions of the sequence over time. The E-value (expect value) is the number of hits that the result received were by chance.

## Maximum Likelihood Phylogeny



## Bayesian Phylogeny




# Sources Cited
