---
title: 'Higher Abundance of Bacteria on Communal Keyboards Compared to Individual Keyboards'
author: "Jacinda Chen"
date: "October 16, 2019"
output: word_document
csl: bioinformatics.csl
bibliography: references.bib
---

```{r setup, include = FALSE}
# load packages for analysis
knitr::opts_chunk$set(echo = FALSE, message = FALSE, warning = FALSE)
```

# Introduction

Computers have become an essential part of college life as a result of the increased dependency on the internet. Therefore, a growing amount of students are required to buy or have access to computers. Food and drinks are constantly consumed around keyboards, which provide resources for creatures living on top of the keyboard, if there are [@malik2014study]. Keyboards are being touched constantly in personal laptops as well as shared computers open to students, faculty and staff (that the university offers). Most people are not aware of the number of microorganisms present on their keyboards [@al2011bacterial]. It is also important to put an emphasis on the importance of human contact as it is generally the cause of microbial migration. Human beings carry different types of microbial groups depending on what a human body part has come into contact with the most. For example, gut and vaginal communities were discovered on toilets [@meadow2014bacterial]. The body part that comes into contact with keyboards the most are hands, which are able to access a wide variety of bacterial communities. Unlike the past, today's society spends a greater amount of time indoors in a built environment, repeatedly in contact with the same surfaces. This allows these same surfaces to accumulate a variety of different microbial organisms associated with humans. Universities are no exception as they make perfect microbial hosts with their large number of inhabitants and numerous communal keyboards [@ross2015microbial]. Since the spread and origin of a large number of bacteria are due to humans, humans should have a responsibility to uphold clean standards in order to minimize risk of infection. Not only do humans contribute to the bacteria on the objects in built and natural environments, humans also increase the amount of airborne bacterial populations indoors, all possessing a possibility of horizontal gene transfer with non-airborne bacteria [@hospodsky2012human]. My primary question was whether or not the keyboards of individual University of San Francisco's (USF's) college students' laptops from the Bioinformatics class have more or less microbacterial species compared to shared computers on campus from the library, the University Center and Toler Hall. Previous research done on this topic has found that shared computers had more microorganisms than individual computers [@anderson2009microbial]. This project will circle around hygiene and aim to figure out if  computers should be cleaned more often and if it would be necessary to raise awareness of keyboard hygiene as most people are not that informed about the number of the microorganisms present on keyboards. It is recommended that food and drink should be separated from computers and keyboards should be disinfected regularly [@hamzeh2011computer]. This is a problem, especially on a school campus where diseases spread quickly. The continuous and widespread transfer of bacteria is a dangerous process as it increases the risk of antibiotic resistance and horizontal transfer among bacteria, which is a huge issue since this hinders the progress of disease treatment [@muniesa2013could]. In the interest of people trying to find ways to prevent the spread of these microorganisms, it is important to keep regularly used computers clean, especially in shared public settings. Although the microorganisms found on computers may not be pathogenic or malignant for most people, they can definitely cause harm for people with compromised immune systems [@fraser2009bacterial]. An example of this could be burn victims, where the issue was raised if computer keyboard contamination was a possible method of transmission of methicillin-resistant *Staphylococcus aurerus* onto patients [@isaacs1998swabbing]. 

Even though different institutions and people can have various methods of cleaning keyboards, without a regular and frequent keyboard cleaning, bacteria is free to grow. The longer that keyboards are not cleaned, the higher the chance that users of the keyboards will contract diseases from the microorganism or transmit the microorganism elsewhere. The microorganisms with resistances to most antibiotics or disinfectants are growing steadily and exponentially, which allows a growing amount of microorganisms to have longer lifespans on keyboards. Because of this, patients with compromised immune systems, like burn victims, no longer have the assurance of antibiotics fully curing them. Most of the bacteria that plague burn victims or other immunosuppressed patients are gram-negative bacteria, which survive significantly longer on hard surfaces (such as polyurethane, or computer keyboard covers) and humid conditions [@neely2000survey]. A longer lifespan gives microorganisms more time to transmit and spread to other places where they have more time to replicate, mutate, and increase survival chances [@eltablawy2009microbial]. Food crumbs, soda residues, and more food residues help millions of bacteria grow. All of the sampled keyboards were positive for microorganism growth [@eltablawy2009microbial]. The environmental surface at which bacteria grow on largely affects their survival times.

My hypothesis is that the keyboards of shared computers have more microbacterial species than individual keyboards because of the increased number of people (students, faculty, staff, etc.) that have access to the keyboard. I collected bacterial samples from three different individual laptops on the "F" key and "A" key from students from USF (University of San Francisco) and three other samples from three public spaces on campus, also on the same keys. The bacteria from the "F" key samples were diluted then grown in cultures with their dilutions, where DNA was extracted from them. DNA was extracted directly from the "A" key samples. Touchdown PCR was done along with Sanger sequencing for the "F" key samples, whose results I analyzed with Geneious Prime and R Studio. Amplicon and Index PCR was done along with Illumina sequencing for the "A" key samples, whose results I analyzed with Bash. My primary findings for the cultured data was a significant difference in the number of bacterial colonies between Communal and Individual computer keyboards. For the molecular data, it was important that the BLAST table agreed with the Maximum Likelihood Phylogeny and the Bayesian Phylogeny in which samples were more closely related. From the culture-free data, I found that there was no significant different between the number of species between communal and individual sample groups *INSERT STUFF*.

# Methods

## Study design

I planned to look at the number of bacteria species on laptop keyboards of individual University of San Francisco students versus the bacteria found on shared computers. I sampled the "F" key and "A" key for three different individual USF students, as well as the keyboards of three different shared computers on the main campus of USF (Toler Hall, University Center, and Gleeson Library). My primary question was whether or not the keyboards of individual USF college students' laptops from the Bioinformatics class have a greater variety of microbacterial species compared to shared computers on campus from the library, the UC and Toler Hall. 

I collected bacteria, diluted the samples, plated the samples on TSA plates, extracted DNA from the samples, did touchdown PCR, gel electrophoresis, and Sanger sequencing. These results were analyzed with Geneious Prime and R Studio.

## Bacterial Collection and Dilution

The surface samples were collected during the lab section of an upper division Bioinformatics class on September 3rd, 2019. In total, twelve tubes were collected. For each location, the "F" key and the "A" key on the keyboard were swabbed. Each sterile swab was dipped in sterile PBS (Phosphate Buffer Saline at pH 7.4) to moisten, then wrung of excess PBS against the side of the tube. I swabbed each surface for 15 seconds. The tip of the swab was broken off into a labelled eppendorf with initials, date,and an unique sample ID. I gathered samples on the first floor of Toler Hall (the shared computer at the front desk), the fourth floor of the University Center (in the graphics center on a shared computer), and the first floor of Gleeson Library (on a shared computer). Subsequently, inside the Bioinformatics class, I gathered samples from three different laptops owned by three different students (Student A, Student B, and Student C) in the same way as the three previous places. The "A" key group was placed in the freezer at -20^o^ Celsius.

## Plating "F" Key Group

A dilution series was done with the "F" key group. I put 200 µL of PBS into the six eppendorfs with swabs in order to do 1:10 dilutions and 1:100 dilutions. Since the type of bacteria was unknown nor how it would grow, it was necessary to dilute the samples in order to maximize the number of environments that the bacteria could possibly grow in. I used 18 100 mm TSA (Tryptic soy agar) plates for plating samples from each of my 2 treatments (n = 3 per group, 6 total). I put 100 µL of an eppendorf from the undiluted series onto a TSA plate and used sterile rattler Plating Beads (5 mm) to spread the medium evenly across the TSA plate. I repeated this step for the other five eppendorfs from the undiluted series, the six eppendorfs from the 1:10 dilution series, as well as six eppendorfs from the 1:100 dilution series. These 18 TSA plates were then incubated at 37^o^ Celsius for a week.

## Cultured DNA Extraction

I followed the manufacturer's protocol from the Sigma REDExtract-N-Amp kit [@weberextract]. I labelled a 1.5 mL tube for each sample and added cells from one colony to the tube using a sterile pipette tip to gently scrape off one colony (not pipetting). One colony was chosen each from the TSA plate of the undiluted sample from Toler Hall, the University Center, Gleeson library, and Student B. Two colonies were chosen from the TSA plate of the undiluted sample from Student C. During DNA extraction, the caustic extraction solution broke down cell walls and the incubation at 95^o^ Celsius was to break membranes. Dilution solution was added to stop the extraction solution from digesting DNA too. The dilution solution was added in 16 minutes from when the extraction solution was first added, which was under twenty minute mark when the extraction solution would have started to break down the DNA as well. Qubit fluorometer was used to measure the concentration of DNA. Qubit samples were incubated in the dark because the Qubit machine is sensitive to the light.

## Culture-Free DNA Extraction

I followed the manufacturer's protocal from the Sigma REDExtract-N-Amp kit [@weberextract]. Extraction solution and dilution solution was added directly into the tube with the swabs from the "A" key. A Qubit fluorometer was used to measure the concentration of DNA.

## Touchdown PCR for Cultured "F" key series

For one reaction, I put 10 µL of Amp, 0.8 µL of 27f primer, 0.8 µL of 1492r primer, 1 µL BSA, and 6.4 µL of water. One µL of water was added for the negative control. PCR conditions were 95^o^ C for 5 minutes, then, 94^o^ C for 30 seconds, 65^o^ C for 30 seconds, 72^o^ C for 1 minute. These last three times were repeated for ten cycles, but the temperature was decreased by one degree per cycle. The next three temperatures and times were 94^o^ C for 30 seconds, 55^o^ C for 30 seconds, 72^o^ C 1 minute. These last three times were repeated for 25 cycles. This was done as touchdown PCR in order to achieve the best quality copies. Lastly, the PCR sample tubes were held at 72^o^ C for 10 minutes and held at 4^o^ C. The electrophoresis gel was made with 2% agarose, SYBR safe dye, 1x TAE buffer and was run at 140 V for 30 minutes. An Ez load 1 kb Molecular Ruler was added to the first and last wells of both lines of wells in the gel. After checking for successful amplification using gel electrophoresis, PCR products were cleaned with ExoSAP (Invitrogen) and sent for unidirectional Sanger sequencing at MCLAB (South San Francisco, CA) using the 27f primer. 

## Analysis

### Molecular Data Analysis

My first task was to assess whether or not the sequences were usable. I used Geneious Prime (version 2019.2.3) to trim or correct usable sequences. This was done by trimming off a large part of the beginning and ending of the sequences (since they were filled with low quality or unusable bases) and going through the low quality  bases in the middle of the sequence to decide whether or not they should be deleted, changed, or an ambiguity code needed to be added from the IUPAC Ambiguity Codes. I used MAFFT to create an appropriate multiple sequence alignment with all five of my usable and corrected sequences with the default settings and an offset value of 0.123 [@katoh2009multiple]. A Maximum Likelihood Phylogeny and a Bayesian Phylogeny were created using *Thermus aquaticus* (NR 025900.1) as an outgroup by downloading the sequence from GenBank [@guindon2005phyml; @huelsenbeck2001mrbayes]. For the Maximum Likelihood Phylogeny, a substitution model of General Time Reversible (GTR) was used with 100 bootstrap replicates with defaults for all other settings. For the Bayesian Phylogeny, I used a substitution model of GTR, a rate variation of Gamma plus Invariants model, as well as a Burn-in Length of 100,000 with all other settings set to default. I used Nucleotide BLAST (Basic Local Alignment Search Tool) by comparing my nucleotide sequences to the sequence databases, where it also calculated the statistical significance of the match [@johnson2008ncbi].

### Bash

INSERT STUFF

### R Studio

I used the packages dplyr and ggplot in order to visualize the data with boxplots. I also tested between different sampling groups using Wilcox statistical tests in order to get the p values.

# Results

```{r load-libraries}
# load libraries
library("ggplot2")
library("dplyr")
library("tidyr")
library("readr")
library("broom")
library("knitr")
```

```{r read-in-culture-data}
# read in the culture data from csv
culture_data <- read_csv(
  "data/raw_data/raw_culture_count_data/BIOL422_culturing_data.csv")
```

## Culture Data

The only TSA plate group of 1x, 10x, and 100x that did not have any culture growing at all was 4F, or the personal laptop from Student A, therefore no data could be drawn from it. 

There was a higher number of colonies that was present on Communal computer keyboards. The median number of colonies from the Communal samples was around 2.5 colonies larger than the median number of around 0 for Individual samples (Figure 1). There was a significant difference between the number of colony abundances between the sample groups (Table 1 Wilcox p = 0.02). Both the Communal and Individual samples had a very small range of the number of morphotypes, with an outlier of two morphotypes (Figure 2). There was no significant difference between the morphotypes between the different places I tested (Table 2 Wilcox p = 0.10). Statistical tests were performed in order to remove human subjectivity with finding statistically significant patterns. Each of the two boxplots are supported with a statistical test for a difference in group means. The easiest approach was to use a Wilcox test (a nonparametric alternative to the t-test).

### Colony Abundance 

The colony abundances from the Communal computers had larger number of colonies compared to the Individual computers, which is shown by the maximum number of colonies observed in the Individual computers remaining smaller than the minimum of the colony abundances from the Communal computers (Figure 1). I also observed a bigger range of larger values for the colony abundances of Communal computers where the minimum and maximum number of colony abundances had a difference of around eight, whereas the differences between the minimum and maximum number of colony abundances for Individual computers was about one. The colony abundance for Individual computers had a singular abnormally abundant colony depicted by the outlier, just under the 2.5 number of colonies mark. A statistical (Wilcox) test was run to compare the number of colony abundances of Communal versus Individual computers at 1x dilution. The mean number of colonies between Communal and Individual computers were significantly different, with a p-value of 0.02 (Table 1; Statistical test for Colony Abundance).

```{r filter-and-plot-abundances}
# filter out only my 1x dilutions and make a boxplot
culture_data %>%
  filter(student_name == "Jacinda Chen") %>%
  filter(dilution_1_10_or_100 == "1") %>%
  ggplot(aes(x = sample_group, y = number_of_colonies)) +
  geom_boxplot() +
  ggtitle(
    "Comparison of Colony Abundances Between Communal and Individual Computers"
    , "1x dilution") +
  xlab("Sample Group") +
  ylab("Number of Colonies")
```

**Figure 1:** Boxplot of colony abundances at Communal versus Individual computers, 1x dilution. There was a higher median number of colonies from Communal Computer samples, and the mean number of colonies were significantly different between the two sites (Wilcox p = 0.02104).

```{r abundances-stats}
# run a statistical test to compare the two groups of abundances
culture_data %>%
  filter(student_name == "Jacinda Chen") %>%
  filter(dilution_1_10_or_100 == "1") %>%
  wilcox.test(data = .,
              number_of_colonies ~ sample_group) %>%
  broom::glance(x = .) %>%
  knitr::kable()
```

**Table 1:** The statistical (Wilcox) test to compare the number of colony abundances between Communal and Individual computer keyboards, 1x dilution.

### Morphotypes

I made a boxplot showing the number of morphotypes in Communal versus Individual computer keyboards for 1x dilution (Figure 2). Both Communal and Individual computer keyboards had an outlier of two morphotypes. The Communal computer keyboards had a median, minimum, and maximum of one morphotype. Individual computer keyboards had a median and minimum of 0 morphotypes and a maximum of 0.75 morphotypes. The mean number of morphotypes between Communal and Individual computer keyboards were not significant with a p-value of 0.10 (Table 2; Statistical test on the number of morphotypes). This was the result of a statistical test to compare the number of morphotypes in Communal versus Individual computer. 

```{r filter-and-plot-morphotypes}
# Comparison of morphotypes between Communal and Individual computer keyboards
culture_data %>%
  filter(student_name == "Jacinda Chen") %>%
  filter(dilution_1_10_or_100 == "1") %>%
  ggplot(aes(x = sample_group, y = as.numeric(number_of_morphotypes))) +
  geom_boxplot() +
  ggtitle(
    "Comparison of Morphotypes In Communal and Individual Computer Keyboards"
    , "1x dilution") +
  xlab("Sample Group") +
  ylab("Number of Morphotypes")
```

**Figure 2:** Boxplot showing the number of morphotypes between Communal and Individual computers, 1x dilution. The mean number of morphotypes were not significantly different between the communal and individual computers (Wilcox p = 0.09679).

```{r morphotype_stats}
# run a statistical test to compare number of morphotypes between two groups
culture_data %>%
  filter(student_name == "Jacinda Chen") %>%
  filter(dilution_1_10_or_100 == "1") %>%
  wilcox.test(data = .,
              as.numeric(number_of_morphotypes) ~ sample_group) %>%
  broom::glance(x = .) %>%
  knitr::kable()
```

**Table 2:** The statistical (Wilcox) test to compare the number of morphotypes between Communal and Individual computer keyboards, 1x dilution.

### Grown Cultures

Two colonies were used from 6F (Figure 3). 3F and 5F both had a single colony growing significantly larger than other colonies. 6F had two different morphotypes, while every other culture had the same morphotype: circular with pale coloring. 2F had many small culture growing. 

![](data/images/method_images/grown_cultures.jpg)

**Figure 3:** A picture showing the usable colonies from culturing with 1F, 2F, 3F, 5F, and 6F sample TSA plates.

## Molecular Data

### Qubit DNA extraction samples of Key "F" series (cultured)

The measured Qubit numbers of the DNA extractions from the "F" key culture samples had a sufficient concentration of DNA in order to ensure that I had enough DNA to amplify in PCR, a later step. All of my samples showed that they had a sufficient amount of DNA from the Qubit numbers, which were all above 5 ng/µL (Table 3).

| Sample | Qubit Number Range (ng/µL) |
|--------|----------------------------|
| 1F     | 5.40                       |
| 2F     | 6.10                       |
| 3F     | 8.76                       |
| 5F     | 7.51                       |
| 6Fo    | 5.66                       |
| 6Fp    | 9.05                       |

**Table 3:** Table of Qubit DNA extraction samples of Key "F" series (cultured)

### Gel Electrophoresis

Bright bands, or higher concentrations, were shown for the 1F, 2F, 3F, 5F, and 6Fp samples, while the 6Fo sample showed a faint band, or a lower concentration (Figure 4). The bands all showed up at a similar place,  using to the ladder. There were no bands at all in the negative control.

![](data/images/gel_images/2019-09-18_BIOL422_Zimmerman_culture_PCRs_Cropped_7.png)

**Figure 4:** Gel electrophoresis image of samples 1F, 2F, 3F, 5F, 6Fo, 6Fp, ladder, and negative control. 

### Sequence Usability Table

The original filename of my sequences came from the Sanger sequencing. There were only five out of the seven sequences that were usable. The number of bases after the sequences were trimmed as well as the number of bases corrected is in Figure 4.

| Sequence Locus | Original Sequence Filename | Corrected Sequence Filename        | Usability | Length after trimming | Manually corrected/adjusted bases |
|----------------|----------------------------|------------------------------------|-----------|-----------------------|-----------------------------------|
| 16S            | JC_1F_27f_A07.ab1          | JC_1F_27f_A07_JC_cleaned_mixed.ab1 | Usable    | 426 bases             | 4                                 |
| 16S            | JC_2F_27f_B07.ab1          | JC_2F_27f_B07_JC_cleaned_mixed.ab1 | Usable    | 658 bases             | 4                                 |
| 16S            | JC_3F_27f_C07.ab1          | JC_3F_27f_C07_JC_cleaned.ab1       | Usable    | 634 bases             | 3                                 |
| 16S            | JC_5F_27f_D07.ab1          | JC_5F_27f_D07_JC_cleaned.ab1       | Usable    | 593 bases             | 7                                 |
| 16S            | JC_6Fo_27f_E07.ab1         | JC_6Fo_27f_E07_JC_failed.ab1       | Unusable  | N/A                   | N/A                               |
| 16S            | JC_6Fp_27f_F07.ab1         | JC_6Fp_27f_F07_JC_cleaned.ab1      | Usable    | 559 bases             | 3                                  |
| 16S            | JC_Neg_27f_G07.ab1         | JC_Neg_27f_G07_JC_failed.ab1       | Unusable  | N/A                   | N/A                               |

**Table 4:** Table depicting the original filenames, usable trimmed sequences that were corrected and renamed, and details surrounding the latter.

### BLAST Table

The high 100% Query Covers for 1F, 2F, 3F, 5F, and 6Fp sequences meant a large percentage of the target sequence covered the query sequence (Table 5). The 5F trimmed sequence had a percent identity of over 98%, showing that the matched BLAST sequence was very similar to my trimmed sequences (Table 5). My trimmed sequence of 1F, 2F, 3F, and 6Fp had a percent identity over 99%, which means the matched BLAST sequences were even more similar. The accession number is an unique tag for a sequence that allows me to track different versions of the sequence over time. The E-value (expect value) was 0.0 for all of my usable sequences, therefore I am confident that the matched BLAST sequences were not due to chance (Table 5). 

| BLAST filenames                    | Description                                                                                         | Percent Identity | Query cover | Accession  | E-Value |
|------------------------------------|-----------------------------------------------------------------------------------------------------|------------------|-------------|------------|---------|
| JC_1F_27f_A07_JC_cleaned_mixed.ab1 | *Staphylococcus epidermidis* strain IBK-11 16S ribosomal RNA gene, partial sequence                   | 99.53%           | 100%        | MN428237.1 | 0.0     |
| JC_2F_27f_B07_JC_cleaned_mixed.ab1 | *Staphylococcus hominis* subsp. novobiosepticus strain IBK-7 16S ribosomal RNA gene, partial sequence | 99.39%           | 100%        | MN428234.1 | 0.0     |
| JC_3F_27f_C07_JC_cleaned.ab1       | *Bacillus subtilis* strain SR3-30 16S ribosomal RNA gene, partial sequence                            | 99.53%           | 100%        | MN421487.1 | 0.0     |
| JC_5F_27f_D07_JC_cleaned.ab1       | *Bacillus subtilis* strain SR3-30 16S ribosomal RNA gene, partial sequence                            | 98.82%           | 100%        | MN421487.1 | 0.0     |
| JC_6Fp_27f_F07_JC_cleaned.ab1      | *Staphylococcus epidermidis* strain LY-2 16S ribosomal RNA gene, partial sequence                     | 99.82%           | 100%        | MH930441.1 | 0.0     |

**Table 5:** Table of nucleotide BLAST results of edited sequences.

### Maximum Likelihood Phylogeny with PhyML

I found that sequences 6Fp and 2F are grouped together in a monophyletic clade with 85 bootstrap support value, while sequences 3F and 5F are grouped together in a monophyletic clade with 100 bootstrap support (Figure 5). This was also supported by the Bayesian Phylogeny (Figure 6).

![](output/phylogenies/sanger_sequences_phyml_phylogeny_image.png)

**Figure 5:** Maximum Likelihood Phylogeny with PhyML using *Thermus aquaticus* as an outgroup.

### Bayesian Phylogeny with Mr. Bayes

For the Bayesian Phylogeny, *Thermus aquaticus* was used as the outgroup. I found that sequences 6Fp and 2F are grouped together in a monophyletic clade, while sequences 3F and 5F are grouped together in a monophyletic clade (Figure 6). This was also supported with the Maximum Likelihood Phylogeny (Figure 5; ML bootstrap > 80).

![](output/phylogenies/sanger_sequences_mrbayes_sorted_topologies_phylogeny_image.png)

**Figure 6:** Bayesian Phylogeny with Mr. Bayes using *Thermus aquaticus* as an outgroup.

```{r read-in-culture-free-data}
# read in the culture-free data from csv
culture_free_data <- read_csv(
  "data/raw_data/raw_culture_free_count_data/BIOL422_culture_free_data.csv")
```

## Culture Free Molecular Data

### Qubit DNA extraction samples of Key "A" series (culture-free)

| Sample | Qubit Number Range (ng/µL) |
|--------|----------------------------|
| 1A     | 5.37                       |
| 2A     | 6.73                       |
| 3A     | 5.95                       |
| 4A     | 6.50                       |
| 5A     | 5.50                       |
| 6A     | 5.96                       |

INSERT TABLE 6

### Illumina Sequence Quality

| Sample | Surviving % | Dropped % |
|--------|-------------|-----------|
| 1A     | 95.16       | 4.84      |
| 2A     | 95.54       | 4.46      |
| 3A     | 95.43       | 4.57      |
| 4A     | 92.90       | 7.10      |
| 5A     | 95.58       | 4.42      |
| 6A     | 95.01       | 4.99      |

INSERT FIGURE 7

### Number of Species per Sample

```{r plot-number-of-species}
# make a boxplot of the number of species per sample
culture_free_data %>%
  ggplot(aes(x = sample_group, y = number_of_species)) +
  geom_boxplot() +
  ggtitle(
    "Comparison of the Number of Different Species Between Sample Groups") +
  xlab("Sample Group") +
  ylab("Species Count")
```

### Wilcox test for the Comparison of Species

```{r abundances-stats}
# run a statistical test to compare the two groups of species numbers
culture_free_data %>%
  wilcox.test(data = .,
              number_of_species ~ sample_group) %>%
  broom::glance(x = .) %>%
  knitr::kable()
```

### Finding BLAST species from Cultured in Culture-Free Data

| Sample | Cultured BLAST             | Culture-Free BLAST |
|--------|----------------------------|--------------------|
| 1A     | *Staphylococcus epidermidis* | Found              |
| 2A     | *Staphylococcus hominis*     | Not Found          |
| 3A     | *Bacillus subtilis*          | Found              |
| 4A     | N/A                          | N/A                |
| 5A     | *Bacillus subtilis*          | Found              |
| 6A     | *Staphylococcus epidermidis* | Found              |

INSERT TABLE 8

### BLAST Table for Control Samples

| Sample Groups            | Most Abundant Species                               | Second Most Abundant Species |
|--------------------------|-----------------------------------------------------|------------------------------|
| Positive Control (zymo1) | *Salmonella enterica* subsp. *enterica serovar Indiana* | *Listeria monocytogenes*       |
| Positive Control (zymo2) | *Salmonella enterica* subsp. *enterica serovar Indiana* | *Listeria monocytogenes*       |
| Positive Control (zymo3) | *Salmonella enterica* subsp. *enterica serovar Indiana* | *Listeria monocytogenes*       |
| Positive Control (zymo4) | *Salmonella enterica* subsp. *enterica serovar Indiana* | *Listeria monocytogenes*       |
| Negative Control 1       | *Acinetobacter tandoii*                               | *Escherichia coli*             |
| Negative Control 2       | *Acinetobacter tandoii*                               | *Micrococcus* sp.              |
| Negative Control 3       | *Acinetobacter tandoii*                               | *Escherichia coli*             |
| Negative Control 4       | *Acinetobacter tandoii*                               | *Haemophilus parainfluenzae*   |

INSERT TABLE 9

# Discussion

My primary question was whether or not the keyboards of individual USF college students' laptops from the Bioinformatics class have more or less microbacterial species compared to shared computers on campus from the library, the UC and Toler Hall. I hypothesized that the keyboards of shared computers have more microbacterial species than individual keyboards because of the increased number of people (students, faculty, staff, etc.) that have access to the keyboard. My primary findings for the culture data was a significant difference in the number of colonies between Communal and Individual computer keyboards. For the molecular data, it was important that the BLAST table agreed with the Maximum Likelihood Phylogeny and the Bayesian Phylogeny in which samples were more closely related.

## Grown Cultures

I found that the 1x dilution cultures worked the best because all of the Communal computer samples had at least one colony and a majority of the Individual computer keyboard samples had at least one colony.

The only TSA plate group of 1x, 10x, and 100x that did not have any culture growing at all was 4F, or the personal laptop from Student A. This was similar to previous research since a majority of the samples contained bacterial contamination and for my small sample size of six different bacterial swabs, five out of six showing bacterial contamination is congruent with previous findings since a majority of samples contained bacterial contamination [@al2011bacterial]. Two colonies were chosen from the TSA plate of the undiluted sample from Student C, or sample 6F, for this reason (Figure 3). 

## Touchdown PCR for "F" key series

The PCR strip tubes were put in the machine for 95^o^ C for 5 minutes. Then, they were put in the machine for 94^o^ C for 30 seconds, 65^o^ C for 30 seconds, and 72^o^ C for 1 minute. These last three cycles were repeated for ten additional cycles, where the temperature was decreased by one degree per cycle. Touchdown PCR was used in order to increase selectivity so that higher quality attractions would click together in the beginning [@korbie2008touchdown]. The purpose of starting at a higher temperature was to be more selective at the first round of 65^o^ C with the copies of the best match. As the temperature decreased by one degree, it created matches and copies preferentially of the already selective primer rather than the original strand. Also, if I had accidentally put two colonies instead of one, one colony would most likely be a better match to the primer, then the primer would preferentially amplify that one.

## Gel Electrophoresis

For the gel electrophoresis, buffer was used instead of water because nothing would happen with water while buffer is able to conduct a high level of current. Agarose gel was used rather than polyacrylamide because agarose is less dense and gives DNA more space to stretch out. The master mix contained glycerol, which made my sample more dense than water, causing it to sink at the bottom of the well in the gel instead of mixing with water in the gel.  There were ladders in the first and last wells of each of the two lines of wells in order for them to act as standards (a bunch of fragments of DNA at known lengths). The ladders act as the positive control from the cyber working. If the electrophoresis and the cyber works, then the known lengths of DNA fragments should be seen. The ladder looks more grouped together above 3 kb because the gel only ran for 30 minutes. There were bright bands shown for 1F, 2F, 3F, 5F, and 6Fp at 1-2 kb, a faint band for 6Fo at 1-2 kb, and no band for the negative control (Figure 4). There were no bands for the negative control because there is no DNA in there, which was what I would expect to see. There was a strange brightness above the bright band possibly because tandem repeats of the ribosome were made due to their importance. In order to fix this problem in future projects, I could change the restriction enzyme, drop the number of cycles (however faint bands would most likely not show at all), or shorten the extension time (to give taq polymerase less time to copy). The fluorescence around 30 kb was due to the gel having auto-fluorescence and was not due to the DNA.

## Geneious Prime

The sequence 6Fo was not usable possibly either because there the DNA was low quality, an incorrect amount of DNA template was used, mixed templates were sequenced, or I used the wrong the primer. For future follow-up experiments, I could use an agarose gel to check the DNA template concentration prior to sequencing or to make my own primer. Knowing that I was sequencing bacteria, some of the ambiguous peaks I observed in the 1F and 2F trimmed sequences were probably because I might have sampled two colonies instead of one (Table 4). Another reason why this may have occurred was because the colony sampled is an entire population, which consists of millions of cells that have divided and might have had mutations. I had over 400 bases for a majority of my sequences after trimming, and five usable sequence (Table 4). This was an acceptable amount of bases to be used for the alignments and phylogenies. All of my usable sequences were mostly high quality 16S sequences that required less than eight bases that needed to be manually corrected (Table 4). 

## Sequence Relationships

The matched BLAST sequences were matched with *Staphylococcus epidermidis*, 2F was *Staphylococcus hominis*, 3F was *Bacillus subtilis*, 5F was *Bacillus subtilis*, and 6Fp was *Staphylococcus epidermidis* (Table 5). This was similar to previously identified bacteria in preceding research set up in the same university conditions [@anderson2009microbial]. The trimmed sequence of 1F was not included in the phylogenies because it was too low quality for an accurate phylogeny. The phylogenetic results using the trimmed sequences of 2F, 3F, 5F, and 6Fp matched with my BLAST results. In both phylogenies, *Thermus aquaticus* was used as the outgroup, from the domain Archaea. Since the bacteria I cultured were unknown, I needed to use the domain Archaea as the outgroup, which had the closest relation to the domain Bacteria. In the Maximum Likelihood Phylogeny, sequences 6Fp and 2F were grouped together in a monophyletic clade with a 85 bootstrap support value, while sequences 3F and 5F were grouped together in another monophyletic clade with a 100 bootstrap support value, which agreed with the separation of genera *Staphylococcus* and *Bacillus* from the BLAST table (Figure 5; Table 5). The Bayesian Phylogeny agreed with the Maximum Likelihood Phylogeny and the BLAST table, resulting with the same groupings (Figure 6). I did not find support for my hypothesis as there was no difference between the number of bacterial species between Communal or Individual computer keyboards. Sequences 6Fp and 2F were from the genus *Staphylococcus*, yet 6Fp was from an Individual Student's computer keyboard while 2F was from a shared computer keyboard (Table 5, Figure 5, Figure 6). Sequences 3F and 5F were from the genus *Bacillus*, yet 3F was from a shared computer keyboard while 5F was from an Individual Student's computer keyboard (Table 5, Figure 5, Figure 6). I did, however, find that there was a significant difference in colony abundance on communal keyboards over individual keyboards (Table 1 Wilcox p = 0.02).

# Sources Cited
